### PR TITLE
snapshot schedule changes (kubernetes and docker) for 1.3

### PR DESCRIPTION
**Pull request by harsh-px**
1) snapshot schedule changes (kubernetes and docker) for 1.3
2) extract common pxctl snap and volume commands in includes

Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

File: scheduler/docker/volume-plugin.md [hunk 1]
```diff
@@ -54,12 +54,31 @@ Options:
...
+
+Scheduled snapshots are only available in Portworx 1.3 and higher.
+
+Use the _snap_schedule_ option to specify the snapshot schedule.
+
+{% include snap-schedule-format.md %}
+
+Some examples of snapshots schedules are:
+
+* snap_schedule="periodic=60,10"
```
Comment (sangleganesh) [inline scheduler/docker/volume-plugin.md]: for inline spec i think "," is to be replaced by "#".
and you can specify multiple schedules together:

snap_schedule="daily=12:00#4;monthly=15@12:00"

File: scheduler/kubernetes/dynamic-provisioning.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Using Storage Classes objects an admin can define the different classes of Portw
...
 | nodes             	| Comma-separated Portworx Node ID's to use for replication sets of the volume                                                                                                                                                                                           	| nodes: "minion1,minion2" 	|
 | aggregation_level 	| Specifies the number of replication sets the volume can be aggregated from                                                                                                                                                                                             	| aggregation_level: "2"   	|
+| snap_schedule     	| Snapshot schedule (PX 1.3 and higher). {% include snap-schedule-format.md %}          	| snap_schedule: periodic=60,10<br><br>snap_schedule: daily=12:00,4<br><br>snap_schedule: weekly=sunday@12:00,2<br><br>snap_schedule: monthly=15@12:00     	|
```
Comment (sangleganesh) [inline scheduler/kubernetes/dynamic-provisioning.md]: this should change to # as well.

Comment (harsh-px) [reply scheduler/kubernetes/dynamic-provisioning.md]: @balapx can you confirm whether you tested `#` or `,` ?

Comment (balapx) [reply scheduler/kubernetes/dynamic-provisioning.md]: I have tested them and it is , and not #.

Logs
-----
[root@minion1 pwx]# /opt/pwx/bin/pxctl -v
pxctl version 1.3.0.0-7958907 (OCI)
[root@minion1 pwx]#

[root@minion1 pwx]# docker volume create --driver pxd --opt snap_schedule="periodic=60#2;daily=12:00#2" --name vol1
Error response from daemon: create vol1: Invalid interval 60#2
[root@minion1 pwx]#
[root@minion1 pwx]# docker volume create --driver pxd --opt snap_schedule="periodic=60,2;daily=12:00,2" --name vol1
vol1

[root@minion1 pwx]# /opt/pwx/bin/pxctl v i vol1
Volume	:  132378361970941807
	Name            	 :  vol1
	Size            	 :  1.0 GiB
	Format          	 :  ext4
	HA              	 :  1
	IO Priority     	 :  LOW
	Creation time   	 :  Feb 28 05:51:29 UTC 2018
	Snapshot        	 :  periodic 1h0m0s,keep last 2, daily @12:00,keep last 2
	Shared          	 :  no
	Status          	 :  up
	State           	 :  detached
	Reads           	 :  0
	Reads MS        	 :  0
	Bytes Read      	 :  0
	Writes          	 :  0
	Writes MS       	 :  0
	Bytes Written   	 :  0
	IOs in progress 	 :  0
	Bytes used      	 :  32 MiB
	Replica sets on nodes:
		Set  0
		  Node 		 :  70.0.34.79 (Pool 0)
	Replication Status	 :  Detached

[root@minion1 pwx]# /opt/pwx/bin/pxctl v l -s
ID			NAME				SIZE	HA	SHARED	ENCRYPTED	COMPRESSED	IO_PRIORITY	SCALE	STATUS
849251589265123185	vol1_daily_2018_Feb_28_06_02	1 GiB	1	no	no		no		LOW		1	up - detached
842124504167349476	vol1_daily_2018_Feb_28_06_03	1 GiB	1	no	no		no		LOW		1	up - detached
1024740037620924686	vol1_periodic_2018_Feb_28_06_02	1 GiB	1	no	no		no		LOW		1	up - detached
916804303417842990	vol1_periodic_2018_Feb_28_06_03	1 GiB	1	no	no		no		LOW		1	up - detached
[root@minion1 pwx]#


With Kubernetes spec file using snap_schedule: "periodic=60#3"

[root@master0 v1]# kubectl describe pvc mysql-pvc-1
Name:          mysql-pvc-1
Namespace:     default
StorageClass:  portworx-sc-repl3
Status:        Pending
Volume:
Labels:        <none>
Annotations:   kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"v1","kind":"PersistentVolumeClaim","metadata":{"annotations":{"volume.beta.kubernetes.io/storage-class":"portworx-sc-repl3"},"name":"mys...
               volume.beta.kubernetes.io/storage-class=portworx-sc-repl3
               volume.beta.kubernetes.io/storage-provisioner=kubernetes.io/portworx-volume
Capacity:
Access Modes:
Events:
  Type     Reason              Age               From                         Message
  ----     ------              ----              ----                         -------
  Warning  ProvisioningFailed  7s (x2 over 15s)  persistentvolume-controller  Failed to provision volume with StorageClass "portworx-sc-repl3": Invalid interval 60#3
[root@master0 v1]#

Comment (sangleganesh) [reply scheduler/kubernetes/dynamic-provisioning.md]: ok that is needed for inline spec i guess, so this should be fine.

**Comment by harsh-px** [comment]
@balapx the build is failing because
```
- ./_site/control/snap.html
  *  internally linking to /control/snapshots-1.2.html, which does not exist (line 1)
     <a href="/control/snapshots-1.2.html">Snapshot 1.2</a>
  *  internally linking to /control/snapshots-1.3.html, which does not exist (line 1)
     <a href="/control/snapshots-1.3.html">Snapshot 1.3</a>
```

**Pull request merged**